### DISCUSSION TITLE
Invitation to Join Quark Organization

### DISCUSSION THREAD

**Discussion by 18z** [quark-engine/quark-engine#178] [General]
Hi all,

Thanks to the hard works done by @Dil3mm3, @3aglew0 and @ciastron. 
We are grateful for your contribution to Quark. 😃 
Hence, we'd like to invite you to join Quark organization. 😄

The invitation link will be sent to you right after this post!
We hope you'll be happy for this. 🕺 

Cheers, 
@18z  

PS. Special thanks to @cryptax for bringing great ppl to this project! 😄

**Comment by 18z** [comment]
Hi @Dil3mm3, @Dil3mm3 and @ciastron 

Can you send a hello to the twitter account of [quark engine](https://twitter.com/quarkengine)?
Cuz we'd like to tag you when we release the new feature. 

Btw, please let me know if you don't want to be tagged. No worries. 😄 

Thanks

**Reply by ciastron** [reply]
Hi @18z , it has been a pleasure for us. I'll send you an hello to twitter !

**Comment by krnick** [comment]
Thank you @Dil3mm3, @3aglew0 and @ciastron for your kind help and contributions to Quark, really appreciate it 🥳

**Comment by 3aglew0** [comment]
Hi @18z, we’re honored to receive your invitation. We are glad to accept it and join Quark organization. However, we’re about to conclude our university semester project related to android malware analysis, so we will not be anymore as active as during this last month. Anyway, we are very interested in Quark and want to see its evolution during time and try to contribute to it in our spare time. Thank you again :smile:

**Reply by 18z** [reply]
Hi @3aglew0, 

No problem my friend! 
 
The most important thing is to have fun through the whole journey. 
We'll keep doing fun things and we expect the unexpected! 

Thanks for the nice works!
Enjoy the vacation between the semesters! 😄
